### PR TITLE
add document for windows

### PR DESCRIPTION
**Pull request by ping-ke**
related issue: https://github.com/ethstorage/es-node/issues/182

es-node can be run in Windows using WSL, add related info to GUIDE.md.

### CODE REVIEW

File: GUIDE.md [hunk 1]
```diff
@@ -255,3 +258,13 @@ By default, when executed, the run.sh script will generate a data file named `sh
 If necessary, you can choose an alternative location for data storage by specifying the full path of the file as the value of the `--storage.files` flag in the run.sh script.
 
 Please refer to [configuration](/README.md#configuration) for more details.
+
+### What can I do about "The zkey file was not downloaded" error?
+
+When you see the following message when running **run.sh**. you can manually download the blob_poseidon.zkey/blob_poseidon2.zkey to `./build/bin/snarkjs/` folder and run it again.
```
Comment (syntrust) [inline GUIDE.md:264]: You should probably tell the user the link to download manually:
https://drive.google.com/file/d/1olfJvXPJ25Rbcjj9udFlIVr08cUCgE4l/view

File: GUIDE.md [hunk 2]
```diff
@@ -162,6 +162,9 @@ nvm use 20
 npm install -g snarkjs
 ```
 
+### Install wsl
```
Comment (qzhodl) [inline GUIDE.md:165]: wsl => WSL

File: GUIDE.md [hunk 3]
```diff
@@ -45,11 +45,11 @@ You can run es-node from a pre-built Docker image, a pre-built executable, or fr
 
 ## From pre-built executables
 
-Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs) and [snarkjs](#install-snarkjs).
+Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs), [snarkjs](#install-snarkjs) and [WSL](https://learn.microsoft.com/en-us/windows/wsl/) for Window. 
 
 Download the pre-built package suitable for your platform:
 
-Linux x86-64 or AMD64:
+Linux x86-64 or AMD64 or Windows:
```
Comment (qzhodl) [inline GUIDE.md:52]: I don't think we have pre-built package for Windows, you may want to mention that for Windows users, you can run Linux version on WSL

File: GUIDE.md [hunk 4]
```diff
@@ -45,11 +45,11 @@ You can run es-node from a pre-built Docker image, a pre-built executable, or fr
 
 ## From pre-built executables
 
-Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs) and [snarkjs](#install-snarkjs).
+Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs), [snarkjs](#install-snarkjs) and [WSL](https://learn.microsoft.com/en-us/windows/wsl/) for Window.
```
Comment (qzhodl) [inline GUIDE.md:48]: => and WSL if you are on Windows

File: GUIDE.md [hunk 5]
```diff
@@ -45,11 +45,11 @@ You can run es-node from a pre-built Docker image, a pre-built executable, or fr
 
 ## From pre-built executables
 
-Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs) and [snarkjs](#install-snarkjs).
+Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs), [snarkjs](#install-snarkjs) and [WSL](https://learn.microsoft.com/en-us/windows/wsl/) if you are on Windows.
```
Comment (qzhodl) [inline GUIDE.md:48]: I would suggest removing the "Install WSL" section below and embed the installing link into the WSL directly

File: GUIDE.md [hunk 6]
```diff
@@ -45,11 +45,11 @@ You can run es-node from a pre-built Docker image, a pre-built executable, or fr
...
-Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs) and [snarkjs](#install-snarkjs).
+Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs), [snarkjs](#install-snarkjs) and [WSL](https://learn.microsoft.com/en-us/windows/wsl/) if you are on Windows. 
 
 Download the pre-built package suitable for your platform:
 
-Linux x86-64 or AMD64:
+Linux x86-64 or AMD64 or WSL(Windows Subsystem for Linux):
 ```sh
 curl -L https://github.com/ethstorage/es-node/releases/download/v0.1.6/es-node.v0.1.6.linux-amd64.tar.gz | tar -xz
 ```
```
Comment (syntrust) [inline GUIDE.md:55]: remove `or AMD64` to avoid ambiguity of `or`
```suggestion
MacOS x86-64
```

File: GUIDE.md [hunk 7]
```diff
@@ -45,15 +45,15 @@ You can run es-node from a pre-built Docker image, a pre-built executable, or fr
 
 ## From pre-built executables
 
-Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs) and [snarkjs](#install-snarkjs).
+Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs), [snarkjs](#install-snarkjs) and [WSL](#install-wsl) if you are on Windows. 
 
 Download the pre-built package suitable for your platform:
 
-Linux x86-64 or AMD64:
+Linux x86-64 or WSL(Windows Subsystem for Linux):
```
Comment (syntrust) [inline GUIDE.md:52]: Suggested a space before `(`
```
WSL (Windows Subsystem for Linux)
```

**Review by syntrust — APPROVED** [review]

File: GUIDE.md (hunk 5 above)
Comment (qzhodl) [reply GUIDE.md]: The issue looks not to be resolved, so reopen it.

**Review by qzhodl — APPROVED** [review]

**Pull request merged**